pkg/controllers/rollout/controller_test.go

```diff
@@ -871,7 +764,8 @@ func TestPickBindingsToRoll(t *testing.T) {
 			},
 			latestResourceSnapshotName: "snapshot-2",
 			crp: clusterResourcePlacementForTest("test",
```

Comment:
Many names of the test cases need improvement. 

i.e. this test case name should be something like 
"test rolling scheduled binding to bound with outdated resources and nil overrides in one shot":